miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/userinfo/TokenData.kt

```diff
@@ -1,10 +1,12 @@
 package com.rakuten.tech.mobile.miniapp.js.userinfo
 
 import androidx.annotation.Keep
+import com.rakuten.tech.mobile.miniapp.permission.AccessTokenScope
 
 /** Access token object for miniapp. */
 @Keep
 data class TokenData(
     val token: String,
-    val validUntil: Long
+    val validUntil: Long,
+    var accessTokenScope: AccessTokenScope = AccessTokenScope()
```

Comment:
Is there any reason to initiate here a new `AccessTokenScope()` object?

Reply:
to not break previous sdk version on `TokenData`.